static/styles/left_sidebar.css

```diff
@@ -464,6 +464,33 @@ li.expanded_private_message {
         position: relative;
         top: 0;
     }
+
+    .group_size {
+        padding: 0 4px;
+        background-color: inherit;
+        margin-left: -1px;
```

Comment:
```
 Visually, I think there's a bit of a horizontal centering issue -- we likely want the presence circles to have their centers horizontally aligned in some reasonable way with these (probably just with the 1-digit shape).
 ```
 To fix this, I have shift it with 1px towards left side using `margin-left: -1px;`